scripts/format.sh

```diff
@@ -29,9 +29,13 @@ function usage() {
   echo >&2 "  -l   Lint check without formatting the source code"
 }
 
+format=false
```

Comment:
I think we should default this to true. Many folks (myself included) are in the habit of just running `./scripts/format.sh`. `-f` becomes a bit of a no-op (which is fine)